### DISCUSSION TITLE
xstate-logger

### DISCUSSION THREAD

**Discussion by codingedgar** [statelyai/xstate#2222] [Show and tell]
Hi!

I made veery simple logger for XState, imitating redux-logger, would love some feedback 💛

https://www.npmjs.com/package/xstate-logger
https://github.com/codingedgar/xstate-logger

Original tweet: https://twitter.com/codingedgar/status/1395052062937845766

**Comment by ChrisShank** [comment]
Is this more for logging to help debug during development or logging is production to help track usage and help debug user issues?

**Reply by codingedgar** [reply]
Would like it to be exactly like redux-logger, but right now its just a pretty print, i was missing a pretty printer for debugging but logging to sentry would be nice.

What are you missing the most? Feel free to open a ticket there if you need something
